packages/input-manager/src/integrations/http/index.ts

```diff
@@ -27,10 +27,25 @@ export class HTTPDevice extends Device {
 
 	async init(): Promise<void> {
 		this.#server = new Server((req, res) => {
-			const triggerId = `${req.method ?? 'GET'} ${req.url}`
-			this.emit('trigger', {
-				triggerId,
-			})
+			if (req.url) {
+				// Example: "https://localhost:8000/my/trigger/path?param0=hey#myHash"
+
+				const triggerArguments: TriggerEventArguments = {}
+				const url = new URL(req.url, 'http://localhost')
```

Comment:
Let's change this to something other than `localhost`. It's confusing on a first glance.

Maybe just 

```suggestion
				const url = new URL(req.url, `http://${req.headers.host}`)
```

Like NodeJS docs suggest for parsing the URL of the request?

Reply:
Good idea! I've made it so, and updated the unit tests since they didn't include the headers